### PR TITLE
Implement includes

### PR DESCRIPTION
**Pull request by liufengyun**
Implement includes

### CODE REVIEW

File: test/jekyll/hydefile.elm [hunk 1]
```diff
@@ -54,6 +54,11 @@ controlflowtags = Regex.replace """\{%\s*if\b((?:(?!%\}.)*)%\}([\s\S]*?)\{%\s*en
       Err msg ->
         """(error: @msg)"""
     )
+  |> Regex.replace """\{% include \s*(.*)\s* %\}""" (\{submatches=[filename]} ->
```
Comment (MikaelMayer) [inline test/jekyll/hydefile.elm]: Could you make the * operator non-greedy? 
 Should be .*?

**Review by MikaelMayer — APPROVED** [review]
Well done !

**Comment by liufengyun** [comment]
Thanks a lot @MikaelMayer !

**Comment by MikaelMayer** [comment]
Github just made it possible to quick commit on a pull request, so it was
very easy for me to just do the fix that I menitoned.
Thanks for your work! I'm looking forward to continue this.

Le lun. 15 avr. 2019 à 13:31, Fengyun Liu <notifications@github.com> a
écrit :

**Pull request merged**